_posts/2018-07-20-snowplow-mini-0.6.0-released/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: Snowplow Mini 0.6.0 released
+title-short: Snowplow Mini 0.6.0
+tags: [snowplow-mini, docker, iglu-server, elasticsearch]
+author: Oguzhan
+category: Releases
+permalink: /blog/2018/07/20/snowplow-mini-0.6.0-released/
+---
+
+We are pleased to announce the 0.6.0 release of Snowplow Mini, our accessible "Snowplow in a box" distribution.
+
+Snowplow Mini is the complete Snowplow real-time pipeline running on a single instance, available
+for easy deployment as a pre-built AMI. Use it to:
```

Comment:
can we mention gcp here, e.g.:

"as a pre-built AMI on AWS as well as a hosted image for GCP"